codemagic.yaml

```
@@ -25,6 +26,32 @@ definitions:
       - secrets
       - emails
   scripts:
+    - &change_build_number_pubspec_ios
+      name: Change the build number in pubspec.yaml for iOS
+      script: | 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)